.cfignore

```diff
@@ -2,5 +2,6 @@
 .hoodie
 .hoodierc
 *.log
+dist
```

Comment:
This is only ignoring the `dist` folder for purposes of pushing to Bluemix. The `dist` folder gets created as part of a build step, right? This build step would happen _after_ the app is pushed to Bluemix, so any `dist` folder that happens to exist in your local development environment should be ignored.